src/app/pages/Collections/Collections.tsx

```diff
@@ -1,3 +1,89 @@
+import { generateClient } from "aws-amplify/data";
+import type { Schema } from "../../../../amplify/data/resource"
+import { useEffect, useState } from "react";
+import { Movie } from "@/app/types/movie-backend-types";
+
+// <ul>{movies.map(movie => <li key={movie.id}>{movie.id}</li>)}</ul>
+
+// const fetchTodos = async () => {
+//   const { data: items, errors } = await client.models.Todo.list();
+//   setTodos(items);
+// };
+
+
 export const CollectionsPage = () => {
-  return <div>CollectionsPage</div>;
+  // const [movies, setMovies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Schema["MovieDB"]["type"][]>([]);
+
+  useEffect(() => {
+    const client = generateClient<Schema>()
+    
+    const fetchMovies = async () => {
+      try {
+        const data = await client.models.MovieDB.list();
+        console.log('data', data);
```

Comment:
Is it this that comes through as undefined?

& can you see data through the amplify console?